build_library/prod_image_util.sh

```diff
@@ -76,7 +77,7 @@ create_prod_image() {
 L+  /etc/ld.so.conf     -   -   -   -   ../usr/lib/ld.so.conf
 EOF
 
-  # Only try to disable rw on /usr if there is a /usr partition 
+  # Only try to disable rw on /usr if there is a /usr partition
```

Comment:
Running `grub_install.sh` at the end of finish_image requires the ESP to be unmounted first. Does it make sense to move that to another part of the process, or change how it works? Avoiding that unmount does remove the need for any disk_util changes.